ipn/ipnlocal/local.go

```diff
@@ -4129,20 +4140,27 @@ func (b *LocalBackend) reloadServeConfigLocked(prefs ipn.PrefsView) {
 // the ports that tailscaled should handle as a function of b.netMap and b.prefs.
 //
 // b.mu must be held.
-func (b *LocalBackend) setTCPPortsInterceptedFromNetmapAndPrefsLocked(prefs ipn.PrefsView) {
+func (b *LocalBackend) setTCPPortsInterceptedFromNetmapAndPrefsLocked(prefs ipn.PrefsView, changed bool) {
```

Comment:
changed isn't documented. do we need it?

Reply:
it depends on whether we remove foreground on save, or on tailscaled-startup. Neither of which are happening, but if on startup, then we need to make sure this happens top level and not in this method. Any preference? 

Either way, I'll push the doc update.

Reply:
Removed in favor of active sessions map.